mail_activity_team/static/src/xml/systray.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates>
 
-    <t t-extend="mail.chat.ActivityMenu">
-        <t t-jquery=".o_mail_navbar_dropdown_channels" t-operation="before">
-            <li class="o_mail_navbar_dropdown_top">
+    <t t-extend="mail.systray.ActivityMenu">
+        <t t-jquery=".o_mail_systray_dropdown_items" t-operation="before">
+            <div class="o_mail_systray_dropdown_top">
                 <div>
-                    <button type="button" class="btn btn-sm o_filter_button active" data-filter='my'> My Activities </button>
-                    <button type="button" class="btn btn-sm o_filter_button" data-filter='team'> Team Activities </button>
+                    <button type="button" class="btn btn-link o_filter_button active" data-filter='my' role="tab"> My Activities </button>
+                    <button type="button" class="btn btn-link o_filter_button" data-filter='team' role="tab"> Team Activities </button>
                 </div>
-            </li>
-        </t>
-        <t t-jquery=".o_notification_counter" t-operation="replace">
-            <span class="o_new_notification_counter badge"/>
+            </div>
         </t>
+<!--        <t t-jquery=".o_notification_counter" t-operation="replace">-->
+<!--            <span class="o_new_notification_counter badge badge-pill"/>-->
+<!--        </t>-->
```

Comment:
to remove?

Reply:
I'm not sure since I don't quite understand why this was needed in the first place. The counter style should be same as it is for the other systray icons IMO. 
Any insight on this @jarroyomorales ?

Reply:
I dont remember why I did this, I guess I had some reason, maybe to test and then forgot to remove it. I have just tried to remove it in 11.0 and it works perfect too.